windows-lb/schema.yaml

```diff
@@ -11,152 +11,362 @@ variableGroups:
+  - vcn_cidr
+  - subnet1_cidr
+  - subnet2_cidr
+
+- title: Load Balancer Configuration
+  visible: 
+    and:
+      - show_advanced
+  variables:
+  - lb_shape
+  - flex_lb_min_shape
+  - flex_lb_max_shape
 
-- title: Advanced Configuration
-  visible: false  
+- title: Load Balancer Configuration
+  visible: 
+    and:
+      - show_advanced
   variables:
-    - num_nodes
-    - instance_ocpus
-    - load_balancer_shape_details_maximum_bandwidth_in_mbps
-    - load_balancer_shape_details_minimum_bandwidth_in_mbps
-    - subnet_cidr
-    - instance_shape
-    - instance_name
-    - availability_domain
-    - userdata
-    - cloudinit_ps1
-    - cloudinit_config
-    - index_html
-    - image_os
-    - image_os_version
-    - is_winrm_configured_for_image
-    - is_winrm_configured_for_ssl
+  - instance_shape
+  - shape_flex_ocpus
+  - shape_flex_memory
+  - instance_name
 
+- title: MS SQL Configuration
+  visible: 
+    and:
+      - show_advanced
+  variables:
+  - mssql_primary_shape
+  - mssql_standby_shape
+
+- title: Hidden Parameters
+  visible: false
+  variables:
+  - userdata
+  - cloudinit_ps1
+  - cloudinit_config
+  - index_html
+  - image_os
+  - image_os_version
+  - is_winrm_configured_for_image
+  - is_winrm_configured_for_ssl
+  - release
 
 outputGroups:
 - title: Deployment Details
   outputs:
-    - instance_public_ip1
-    - instance_public_ip2
-    - username1
-    - username2
-    - lb_public_ip
+  - instance_public_ip1
+  - instance_public_ip2
+  - username1
+  - username2
+  - lb_public_ip
 
 outputs:
+
   instance_public_ip1:
     type: link
     title: Instance 1
     description: Public IP for connecting to instance 1 over Remote Desktop
+  
   instance_public_ip2:
     type: link
     title: Instance 2 Public IP
     description: Public IP for connecting to instance 2 over Remote Desktop
+  
   username1:
     type: link
     title: Username (instance 1)
     description: User name login to instance 1
+  
   username2:
     type: link
     title: Username (instance 2)
     description: Username to login to instance 2
+  
   lb_public_ip:
     type: link
     title: Launch Application
     description: Open the ASP.NET application deployed to OCI
 
-primaryOutputButton: lb_public_ip
-
 variables:
+
   instance_password:
     type: password
     required: true
     title: "Windows login password for user: opc"
     description: "Used to login to the instance over remote desktop. Password must at least 12 characters long and include at least one uppercase letter, one lowercase letter, and one numeric character"
     pattern: "^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[!@#$%^&*()_+\\-=\\[\\]{};':\\\"\\\\|,.<>\\/?]).{12,32}$"
     visible: true
+ 
   compartment_ocid:
     type: oci:identity:compartment:id
     title: "Compartment"
     description: "The compartment in which to create Apex instance"
     required: true
-  load_balancer_shape_details_maximum_bandwidth_in_mbps:
+  
+  availablity_domain_name:
```

Comment:
Ideally this does not need to be exposed since we can figure out it th selected compute shapes are available in a random AD.

Reply:
@jeevanjoseph again the same topic. I am opting towards the choice of the customer, but we can discuss it.